lib/sentry/event.ex

```diff
@@ -191,4 +199,23 @@ defmodule Sentry.Event do
 
   defp arity_to_integer(arity) when is_list(arity), do: Enum.count(arity)
   defp arity_to_integer(arity) when is_integer(arity), do: arity
+
+  defp is_in_app?(nil, _in_app_whitelist), do: false
+  defp is_in_app?(_, []), do: false
```

Comment:
In the case that no lines are marked as "in_app", it just shows everything with no option to filter by "in_app".

File path is probably a no go, as it doesn't give us full path. Example:

```elixir
{Poison.Encoder.Any, :encode, 2, [file: 'lib/poison/encoder.ex', line: 383]}
{Poison, :encode!, 2, [file: 'lib/poison.ex', line: 41]}
```